docs/source/development.rst

```diff
@@ -25,9 +25,14 @@ libraries. On Debian or Ubuntu, we can install these using
 
     $ sudo apt-get install python-dev zlib1g-dev git libxslt1-dev
 
-.. note::
-    TODO: Document this basic step for other platforms? We definitely want
-    to tell people how to do this with Brew or ports on a Mac.
+Alternatively, on Mac computers, install Anaconda Python <http://continuum.io/downloads> by following the installation instructions presented on the Continuum download page.
```

Comment:
in vim, this can be done with the `gq` command after selecting the lines in visual mode or whatever